MANIFEST.in

```diff
@@ -5,10 +5,11 @@ include LICENSE
 include README.rst
 include derex/runner/restore_dump.py.source
 
-recursive-include derex/runner/templates *
+recursive-include derex/runner/cli *
```

Comment:
Isn't this already implied, being the first case in [this list](https://packaging.python.org/guides/using-manifest-in/#how-files-are-included-in-an-sdist)?

> The following files are included in a source distribution by default:
> * all Python source files implied by the py_modules and packages setup() arguments

Reply:
You're right ! The problem was in the setuptools `packages` argument.